controller/handler/util_test.go

```diff
@@ -52,10 +55,15 @@ func TestExtractPageAndPageSizeFromRequest(t *testing.T) {
 			ExpectedPageSize: DefaultPageSize,
 		},
 	}
+	storageCfg := &storage.Config{}
+	storageCfg.ValidateAndSetDefaults()
+	cfg := &config.Config{
+		Storage: storageCfg,
+	}
 	for _, scenario := range scenarios {
 		t.Run("page-"+scenario.Page+"-pageSize-"+scenario.PageSize, func(t *testing.T) {
 			request, _ := http.NewRequest("GET", fmt.Sprintf("/api/v1/statuses?page=%s&pageSize=%s", scenario.Page, scenario.PageSize), http.NoBody)
-			actualPage, actualPageSize := extractPageAndPageSizeFromRequest(request)
+			actualPage, actualPageSize := extractPageAndPageSizeFromRequest(request, cfg.Storage.MaximumNumberOfResults)
```

Comment:
Is there really a need to initialize a storage config here since we're just passing `cfg.Storage.MaximumNumberOfResults`?

I think we'd be better off removing 
```
storageCfg := &storage.Config{}
	storageCfg.ValidateAndSetDefaults()
	cfg := &config.Config{
		Storage: storageCfg,
	}
```
adding `MaximumNumberOfResults` to the scenario struct definition and just doing
```
actualPage, actualPageSize := extractPageAndPageSizeFromRequest(request, scenario.MaximumNumberOfResults)
```
That way we can properly test the behavior of `extractPageAndPageSizeFromRequest` with different `MaximumNumberOfResults` values.